src/services/azure/constants.rs

```diff
@@ -0,0 +1,15 @@
+// Env values used in azure services.
+//AZURE_STORAGE_ACCOUNT = Credential.access_name
+#[allow(dead_code)]
+pub const AZURE_STORAGE_ACCOUNT: &str = "AZURE_STORAGE_ACCOUNT";
+//AZURE_STORAGE_ACCOUNT = Credential.shared_key
+#[allow(dead_code)]
+pub const AZURE_STORAGE_KEY: &str = "AZURE_STORAGE_KEY";
+#[allow(dead_code)]
+pub const CONTENT_MD5: &str = "Content-MD5";
```

Comment:
Please use `content-md5` to represent the header name which adheres to hyper's standard header name.